src/Sulu/Bundle/CategoryBundle/Behat/CategoryContext.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+/*
+ * This file is part of the Sulu CMS.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\CategoryBundle\Behat;
+
+use Sulu\Bundle\TestBundle\Behat\BaseContext;
+use Behat\Behat\Context\SnippetAcceptingContext;
+use Sulu\Bundle\CategoryBundle\Entity\Category;
+
+/**
+ * Behat context class for the CategoryBundle
+ */
+class CategoryContext extends BaseContext implements SnippetAcceptingContext
+{
+    protected function getCategoryManager()
```

Comment:
Also protected methods should always be at the bottom of the class, follwed by private.